eo-maven-plugin/src/main/java/org/eolang/maven/Home.java

```diff
@@ -36,140 +36,148 @@
 import org.cactoos.io.TeeInput;
 import org.cactoos.scalar.IoChecked;
 import org.cactoos.scalar.LengthOf;
-import org.cactoos.text.TextOf;
 
 /**
- * Save a file operation.
+ * Class for saving and loading files.
  *
- * @since 0.1
+ * @since 0.27
+ * @todo #1105:30min create load function (it has to be able read by path)
+ *  It should be able to load data from file
+ *  We also need to add new unit test
  */
-public final class Save {
-
-    /**
-     * Content.
-     */
-    private final Input content;
-
-    /**
-     * Path.
-     */
-    private final Path path;
-
-    /**
-     * Ctor.
-     * @param input The input
-     * @param file File to save to
-     */
-    public Save(final InputStream input, final Path file) {
-        this(new InputOf(input), file);
-    }
-
-    /**
-     * Ctor.
-     *
-     * @param txt The content
-     * @param file The path
-     */
-    public Save(final String txt, final Path file) {
-        this(new TextOf(txt), file);
-    }
-
+@SuppressWarnings("PMD.TooManyMethods")
+public class Home {
     /**
-     * Ctor.
-     *
-     * @param txt The content
-     * @param file The path
+     * Current working directory.
      */
-    public Save(final Text txt, final Path file) {
-        this(new InputOf(txt), file);
-    }
+    private final Path cwd;
 
     /**
      * Ctor.
-     *
-     * @param bytes The content
-     * @param file The path
      */
-    public Save(final byte[] bytes, final Path file) {
-        this(new InputOf(bytes), file);
+    Home() {
+        this(Paths.get(""));
     }
 
     /**
      * Ctor.
-     * @param input The input
-     * @param file File to save to
-     */
-    public Save(final Input input, final Path file) {
-        this.content = input;
-        this.path = file;
-    }
-
-    /**
-     * Save the file to the path, without any checked exceptions.
+     * @param path Path
+     * @todo #1105:30min we need to modify function. It has to change
+     *  path by replacing bad characters (not utf-8) or maybe
+     *  we can invent any other solution
      */
-    public void saveQuietly() {
-        try {
-            this.save();
-        } catch (final IOException ex) {
-            throw new IllegalStateException(ex);
-        }
+    Home(final Path path) {
```

Comment:
Where are you planning to use this constructor? rn it's used only in the default constructor and in the test, didn't see any other usages
Or you are going to use it in other places after solving the puzzle?

Reply:
Yes, we will use it. We decided to add it with Egor